website/docs/TestingEnvironment.md

```diff
@@ -0,0 +1,160 @@
+---
+id: testing-env
+title: Testing Environment
+---
+
+## Testing Environment
```

Comment:
I read through all of that and it's great deep dive into how things work under the hood! However, this is definitely for more advanced users, or when someone needs to debug the problems in this library or their tests. This gives me a hint, that we should be very explicit about it. And either add a note, like React docs does in many places, or make it, idk, a contributing or advanced section :)